src/Raven.Server/Documents/ETL/Providers/Elasticsearch/Handlers/ElasticSearchEtlConnectionHandler.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+using Nest;
+using Newtonsoft.Json;
+using Raven.Client.Documents.Operations.ETL.ElasticSearch;
+using Raven.Server.Routing;
+using Raven.Server.Web;
+using Raven.Server.Web.System;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Authentication = Raven.Client.Documents.Operations.ETL.ElasticSearch.Authentication;
+
+namespace Raven.Server.Documents.ETL.Providers.ElasticSearch.Handlers
+{
+    public class ElasticSearchEtlConnectionHandler : RequestHandler
+    {
+        [RavenAction("/admin/etl/elasticsearch/test-connection", "POST", AuthorizationStatus.Operator)]
+        public async Task GetTestSqlConnectionResult()
+        {
+            string url = GetQueryStringValueAndAssertIfSingleAndNotEmpty("url");
+            string authenticationJson = await new StreamReader(HttpContext.Request.Body).ReadToEndAsync();
+            Authentication authentication = JsonConvert.DeserializeObject<Authentication>(authenticationJson);
+
+            ElasticClient client = ElasticSearchHelper.CreateClient(new ElasticSearchConnectionString { Nodes = new[] { url }, Authentication = authentication });
+
+            PingResponse pingResult = await client.PingAsync();
```

Comment:
I think it should be wrapped by `try / catch` same like in SQL ETL:

https://github.com/ravendb/ravendb/blob/v5.2/src/Raven.Server/Documents/ETL/Providers/SQL/Handlers/SqlEtlHandler.cs#L44-L59

Reply:
done